### PR TITLE
Allow specifying sector in search form backend

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1816)

### CODE REVIEW

File: company/forms.py [hunk 1]
```diff
@@ -22,6 +22,11 @@ class CompanySearchForm(forms.Form):
         widget=forms.HiddenInput,
         initial=1,
     )
+    sector = forms.ChoiceField(
+        required=False,
+        widget=forms.HiddenInput,
+        choices=choices.COMPANY_CLASSIFICATIONS,
```
Comment (kowalcj0) [inline company/forms.py]: @richtier does it mean that the Classifications will show up in the "suggestions/auto-completion"?

Comment (richtier) [inline company/forms.py]: @kowalcj0 currently, no UI change will happen as a result of this change - this is purely backend change.

The reason for that is we don't have the designs yet for the UI.

note though that the form will reject an invalid sector - if the user specifies a sector that is not in `choices.COMPANY_CLASSIFICATIONS` then a validation error will occur.

Comment (kowalcj0) [inline company/forms.py]: @richtier what if user specifies a correct keyword and a wrong sector? will such search work?

Comment (richtier) [inline company/forms.py]: @kowalcj0 nope - it will result in a validation error.

**Pull request merged**